stories/module-analytics.stories.js

```diff
@@ -124,6 +127,42 @@ storiesOf( 'Analytics Module', module )
 			</WithTestRegistry>
 		);
 	} )
+	.add( 'Property Select including GA4 properties', () => {
+		const { accounts, properties, profiles } = fixtures.accountsPropertiesProfiles;
+		/* eslint-disable sitekit/acronym-case */
+		const accountID = properties[ 0 ].accountId;
+		const propertyID = profiles[ 0 ].webPropertyId;
+		/* eslint-enable */
+		const setupRegistry = ( { dispatch } ) => {
+			dispatch( STORE_NAME ).receiveGetAccounts( accounts );
+			dispatch( STORE_NAME ).finishResolution( 'getAccounts', [] );
+
+			// eslint-disable-next-line sitekit/acronym-case
+			dispatch( STORE_NAME ).receiveGetProperties( properties, { accountID: properties[ 0 ].accountId } );
+			dispatch( STORE_NAME ).receiveGetProfiles( profiles, {
+				accountID,
+				propertyID,
+			} );
```

Comment:
I believe we don't need profiles for this story, do we? Let's get rid of it if we don't need it.

Reply:
If I comment it out then `select( MODULES_ANALYTICS_4 ).hasFinishedResolution( 'getProperties', [ accountID ] );` is always `false`